commodore/helpers.py

```diff
@@ -170,9 +170,6 @@ def relsymlink(srcdir, srcname, destdir, destname=None):
     # https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.relative_to
     link_src = os.path.relpath(P(srcdir) / srcname, start=destdir)
     link_dst = P(destdir) / destname
-    try:
-        if link_dst.exists():
-            os.remove(link_dst)
-        os.symlink(link_src, link_dst)
-    except Exception as e:
-        raise click.ClickException(f"While setting up symlinks: {e}") from e
```

Comment:
As far as I can tell, the other call sites of `relsymlink` rely on the ClickException being thrown here. Note that the `ClickException` produces an user-friendly error message when it propagates all the way to the top, while other exceptions generate rather unpleasant error messages.

Removing the exception handling here requires that exception handling is introduced at call sites of `relsymlink` (directly or indirectly) to avoid having bare exceptions propagated all the way to the user.

Reply:
After discussing with @chrisglass directly, we've decide to let the exceptions bubble up for other call sites and introduce good error messages when such a case pops up.

Reply:
Thanks a lot!